tests/vmi_multus_test.go

```diff
@@ -271,7 +271,13 @@ var _ = Describe("Multus Networking", func() {
 
 			By("checking virtual machine instance has two interfaces")
 			checkInterface(vmiOne, "eth0", "#")
-			checkInterface(vmiOne, "eth1", "#")
+			checkInterface(vmiOne, "net1", "#")
```

Comment:
Interesting. So introducing ```cloud-init```, we now rename SR-IOV interfaces inside VMI. This is change in behaviour. What's the justification to go with it? Can we stick to the old names? This should be included in release notes at the very least.

Reply:
@greg-bock one may still configure it statically inside VMI though. But yes, perhaps a release note is an overreaction at this point. :)

Reply:
> @greg-bock one may still configure it statically inside VMI though. But yes, perhaps a release note is an overreaction at this point. :)

Yes (although you run into similar issues with bios naming depending on the image and driver).

If you pass something in networkData it will not run discovery so you can just do "#" in the spec to disable it (I will note this in the docs). I was/am tempted to be able to include all devices, set a top level default and allow interface specific overrides but for a first pass it seemed like overkill. The current functionality doesn't change past behavior and seemed less intrusive. Even resolv.conf configured via cloud-init will likely be overwritten by dhcp if you have bridged interfaces in the VM.